cpp/src/pypowsybl.cpp

```diff
@@ -418,6 +418,19 @@ void updateNetworkElementsWithStringSeries(void* network, element_type elementTy
                 idPtr.get(), valuePtr.get(), elementCount);
 }
 
+void newElement(void* network, const std::string& id, element_type elementType,
```

Comment:
@yichen88 have you tried to create Series from Python side instead of having all theses maps?

Reply:
When we add elements, we need a `ROW` of data to build an equipment together, we can not do with `COLUMN'` in Series.